Library/Formula/lfe.rb

```diff
@@ -0,0 +1,30 @@
+require "formula"
+
+class Lfe < Formula
+    homepage "http://lfe.io/"
+    url "https://github.com/rvirding/lfe/archive/v0.9.1.tar.gz"
+    sha1 "015c23e7c761c2ccfefb7a9f97f85fe09f1cd648"
+
+    head "https://github.com/rvirding/lfe.git", :branch => "develop"
+
+    depends_on "erlang"
+    depends_on "rebar"
+
+    def install
+        system "rebar compile"
+        bin.install Dir['bin/*']
```

Comment:
no, erlang folk like releases ;-).

The lfe author is working on a revamped installation approach, but this will take a wee while.